cli/src/config.rs

```diff
@@ -17,9 +17,18 @@ To learn more, checkout https://apollo.dev/cli/telemetry\n";
 #[derive(Debug, Serialize, Deserialize)]
 pub struct CliConfig {
     pub machine_id: String,
+    pub api_key: Option<String>,
```

Comment:
instead of `api_key`, how about `personal_api_key` since we do have service tokens as well?

Reply:
In ci people would use service tokens to push/interact with the cli. I don't think we should differentiate here whether it is a personal or service token, since we don't have a separate workflow for authenticating as a service and authenticating as a user.